golang/internal/grpc/grpc.go

```diff
@@ -591,41 +638,65 @@ func executeSecretList(
 	}
 }
 
-func executeUpdate(ctx context.Context, command *agent.AgentUpdateRequest, updateFunc SelfUpdateFunc) {
+func executeUpdate(loop *ClientLoop, command *agent.AgentUpdateRequest, updateFunc SelfUpdateFunc) {
 	if updateFunc == nil {
 		log.Error().Msg("Self update function not implemented")
 		return
 	}
 
-	err := updateFunc(ctx, command.Tag, command.TimeoutSeconds)
+	_, err := config.ValidateAndCreateJWT(command.Token)
 	if err != nil {
-		log.Error().Stack().Err(err).Msg("Update error")
+		log.Error().Msg("Jwt validation failed")
 
-		errorString := err.Error()
-		resp := &agent.AgentAbortUpdate{
-			Error: strings.ToUpper(errorString[0:1]) + errorString[1:],
-		}
+		abortUpdate(loop.Ctx, err)
+		return
+	}
 
-		_, err := grpcConn.Client.AbortUpdate(ctx, resp)
-		if err != nil {
-			log.Error().Stack().Err(err).Msg("Update abort request error")
-		}
+	err = loop.executeReplaceToken(&agent.ReplaceTokenRequest{
+		Token: command.Token,
+	})
+
+	if err == nil {
```

Comment:
We are not using onelines for error handling? @nandor-magyar 
```
if err := updateFunc(loop.Ctx, command, updateOptionsFromAppConfig(loop.AppConfig)); err != nil {
	abortUpdate(loop.Ctx, err)
}
```

Reply:
If there err is already defined, a shadow warning is shown, that would make the lint fail, so it is fine. I don't we need a rule for this, both forms are correct, whichever is more readable is to be used.